canonical-web-and-design/snapcraft.io
# Connect to repo

## Done

-  Add connect snap to repo functionality
- Update styling for Builds

## Issue / Card

Fixes #2484 
Fixes #2485 

## QA

- Pull the branch
- Run the site using the command `./run`
- View the site locally in your web browser at: http://0.0.0.0:8004/[snap_name]/builds
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check a snap with builds and notice the new position of the `Trigger new build` button
- Check a snap that is not connected to any repos (both with JS on and disabled) and see that you can connect the snap to your repo and trigger a new build

## Screenshots

[if relevant, include a screenshot]
